codalab/worker/default_bundle_manager.py

```diff
@@ -34,6 +34,14 @@ def _schedule_run_bundles(self):
         self._schedule_run_bundles_on_workers(workers, user_owned=True)
         self._schedule_run_bundles_on_workers(workers, user_owned=False)
 
+    def _fail_on_excess_resource(self, value, user_fail_string=None, global_fail_string=None, user_max=None, global_max=None):
```

Comment:
can you choose a different name for this fn? Sounds almost the same as the one below

Reply:
will do 👍